draft-ietf-tls-subcerts.md

```diff
@@ -489,16 +489,16 @@ delegated credentials MUST terminate the connection with an
 
 On receiving a delegated credential and a certificate chain, the peer
 validates the certificate chain and matches the end-entity certificate to the
-peer's expected identity.  It also takes the following steps:
-
-1. Validate that DelegatedCredential.cred.valid_time is no more than 7 days.
-1. Verify that the current time is within the validity interval of the credential.
-   This is done by asserting that the current time is no more than the
-   delegation certificate's notBefore value plus DelegatedCredential.cred.valid_time.
-1. Verify that the delegated credential's remaining validity time is no more than the maximum validity
-   period. This is done by asserting that the current time is no more than the delegation
-   certificate's notBefore value plus DelegatedCredential.cred.valid_time plus
-   the maximum validity period.
+peer's expected identity.  It also takes the following steps, with the expiry
```

Comment:
How about something like this:
```
Define `expiry time` as the delegation certificate's notBefore value plus DelegatedCredential.cred.valid_time. Then perform the following checks:
```

Reply:
Thanks! I'll update to use that phrasing.